webapp/app/partials/perf/perfctrl.html

```diff
@@ -35,8 +35,11 @@
   <div id="graphs">
     <select ng-model="myTimerange" ng-options="timerange.text for timerange in timeranges track by timerange.value" ng-change="timeRangeChanged()">
     </select>
-    <hr/>
     <div id="graph"></div>
+    <div id="graph-bottom">
```

Comment:
I think we should make displaying this conditional on seriesList have a length greater than 0. Show be able to do this with ngShow: https://docs.angularjs.org/api/ng/directive/ngShow